src/Http/Response.php

```diff
@@ -925,6 +925,9 @@ public function getStatusCode()
      */
     public function withStatus($code, $reasonPhrase = '')
     {
+        if (!isset($this->_statusCodes[$code])) {
+            throw new InvalidArgumentException('Unknown status code');
```

Comment:
You need to add `use InvalidArgumentException` at top of file.

Reply:
@ADmad It's already present in the file at rule 27.

Reply:
@tuna-rootnet Sorry, I missed that.

Reply:
Can we the user what the invalid value was? And perhaps what the acceptable range is?